experiments/pages/pageSupportRemover/README.md

```diff
@@ -0,0 +1,9 @@
+# Page Support File Remover
+
+As of Skuid release 12.2.0, page support files are no longer needed by Skuid to render in-line Javascript and CSS. This is not a breaking change and orgs with page support files will continue to work as expected. However, if users want to keep their static resources clean, this tool will attempt to find any page support files and then allow users to remove them.
+
+## Usage
+
+To use this tool, create a Skuid page in your Salesforce org with the page XML stored in this folder. Once created, click **Load all Page Support Files**, select the files you wish to be deleted, and then click **Delete Page Support Files**
+
+Using this tool does not delete your inline Javascript or CSS, it only removes the page support file generated by Skuid when the page was created.
```

Comment:
Comma splice, plus there are other instances when page support files may be generated, i.e. by the user manually.

```suggestion
Using this tool does not delete your inline Javascript or CSS. It only removes the page support file generated by Skuid.
```